apps/web/vibes/soul/primitives/toaster/index.tsx

```diff
@@ -32,7 +31,7 @@ export const Toaster = ({ ...props }: ToasterProps) => {
 };
 
 export const toast = {
-  success: (message: ReactNode, options?: ToastOptions) => {
```

Comment:
There's a reason why we had support for `ReactNode` here. I think we need to make sure we look at usage in Catalyst before breaking the APIs. I'm pretty sure Catalyst relies on being able to pass a `ReactNode` here. I think we can change the API so that not everything takes a `ReactNode`, but as it is right now, this would break Catalyst